### ISSUE TITLE
terraform taint null_resource doesn't work anymore (v0.7.x).

### ISSUE THREAD

**Issue by lesaux** [hashicorp/terraform#8032]
It seems `terraform taint null_resource` doesn't work in v0.7.1 anymore (I'm using Terraform v0.7.1-dev).

```
resource "null_resource" "some_command" {

    provisioner "remote-exec" {
        connection {
            host = "somehost"
            user = "someUser"
        }
        inline = [
          "sudo docker run -d --restart=always -p 8080:8080 someimage
        ]
    }

}
```

```
$ terraform taint null_resource.some_command
The resource null_resource.some_command in the module root has been marked as tainted!
$ terraform plan
...
null_resource.some_command: Refreshing state...
...
No changes. Infrastructure is up-to-date. This means that Terraform
could not detect any differences between your configuration and
the real physical resources that exist. As a result, Terraform
doesn't need to do anything.
```

**Comment by afritzler** [comment]
Same issue here. I am on terraform 0.7.0.

**Comment by phinze** [comment]
Thanks for the report! This is fixed in #8167 which will come out with v0.7.1 due out next week. 👍
